examples/ChuaCircuit/sim_and_plot.py

```diff
@@ -44,10 +60,40 @@
     MODELS, RESULTS_DIR = write_script(EXPERIMENTS, working_dir=WORKING_DIR,
                                        packages=PACKAGES, fname=FNAME)
 
-    # Ask Dymola to run the script.
-    os.system('dymola ' + FNAME) # For Linux
-    # TODO: Add support for Windows.
-    # os.system(r'C:\Program files\Dymola\bin\Dymola.exe ' + FNAME) # For Windows
+    dymola = None
+
+    def get_last_log(dymola):
+        log = dymola.getLastErrorLog()
+        print(log)
+
+    try:
+        # Instantiate the Dymola interface and start Dymola
+        dymola = DymolaInterface()
+
+
+        # dymola.ExecuteCommand("Advanced.CompileWith64=2")
+        result = dymola.ExecuteCommand('Modelica.Utilities.System.setWorkDirectory("' + Path(WORKING_DIR).as_posix() + '")')
```

Comment:
Does this achieve something different than simple `dymola.cd(Path(WORKING_DIR).as_posix())`?